functions/rust-mnist/serverless.yaml

```diff
@@ -0,0 +1,34 @@
+service: rust-examples
+configValidationMode: off
+useDotenv: true
+provider:
+  name: scaleway
+  scwToken: ${env:SCW_SECRET_KEY}
+  scwProject: ${env:SCW_DEFAULT_PROJECT_ID}
+  runtime: rust165
+  scwRegion: ${env:SCW_DEFAULT_REGION}
```

Comment:
We usually don't need to specify the environment variables here for provider credentials, they can be read from the CLI config file (although in this case they're used in the functions themselves so it's not such a big deal).